app/styles/partials/footer.scss

```diff
@@ -33,3 +33,68 @@ footer {
     right: 0 !important;
   }
 }
+
+
+@media screen and (min-width : 1200px) {
+    #main-container {
```

Comment:
@niranjan94 We can use SASS placeholder to put it up in a place and then just adding class to the parent of the rails would balance the rails and the middle part for any pages.